File: TFG_Back/Models/Database/Entidades/Customer.cs
namespace TFG_Back.Models.Database.Entidades
{
    // Entidad que representa a un cliente en la base de datos.
    public class Customer
    {
        public int Id { get; set; }
        public int CIF { get; set; }
        public string Name { get; set; }
        public string Adress { get; set; }
        public int PostalCode { get; set; }
        public string PlaceOfResidence { get; set; }
        public int PhoneNumber { get; set; }
        public string Email { get; set; }
        public string AdminEmail { get; set; }

        // Clave foránea para la relación con PaymentMethod.
        public int PaymentMethodId { get; set; }
        // Propiedad de navegación para acceder al método de pago asociado.
        public PaymentMethod PaymentMethod { get; set; }
    }
}

File: TFG_Movil/app/src/main/java/com/example/tfg_movil/model/customer/Customer.kt
package com.example.tfg_movil.model.customer

// Modelo de datos para lo que será un Paciente
data class Customer(
    val id: Int,
    val cif: Int,
    val name: String,
    val adress: String,
    val postalCode: Int,
    val placeOfResidence: String,
    val phoneNumber: Int,
    val email: String,
    val adminEmail: String,
    val paymentMethodId: Int
)

